src/components/molecules/ChatsList/ChatsList.tsx

```diff
@@ -75,6 +75,8 @@ const ChatsList: React.FunctionComponent = () => {
 
   const onClickOnSender = useCallback(
     (sender: WithId<User>) => {
+      if (!privateChats) return;
+
       const chatsToUpdate = privateChats.filter(
         (chat) => !chat.isRead && chat.from === sender.id
       );
```

Comment:
Looking at the rest of the logic in this `useCallback`, do we want to bail out early? That would mean `setSelectedUser` never gets called?

Also, for cleanup, we can do the `chatsToUpdate` `filter` -> `map` in a chain without extracting the `const`, and since we aren't doing anything with the return value, it's probably better to use `forEach` or similar instead of `map`

Reply:
Hm, maybe I rushed this fix. I believe privateChats is only not set when data hasn’t been loaded. So it doesn’t need to be functional in that situation.

Reply:
And yes to these other cleanups for sure. Was definitely rushed getting in here.